src/Committee/CommitteeManager.php

```diff
@@ -479,4 +479,34 @@ public function getCommitteeSupervisor(Committee $committee): ?Adherent
     {
         return $this->getMembershipRepository()->findSupervisor($committee->getUuid()->toString());
     }
+
+    public function getCommitteesAndMembersByCoordinates(
+        Coordinates $coordinates,
+        int $count = self::COMMITTEE_PROPOSALS_COUNT
+    ): array {
+        $committees = $this->getCommitteeRepository()->findNearbyCommittees($coordinates, $count);
+
+        return self::getFormattedCommitteesAndMembers($committees);
```

Comment:
should be `$this->` instead of `self::` (this one is for static call) 
I may be wrong, please let me know :D